src/components/inputs/markdown/markdown-wrapper.component.tsx

```diff
@@ -6,7 +6,17 @@ const MarkdownWrapper: React.FC<{ markdown: string | string[] }> = ({ markdown }
     <ReactMarkdown
       children={Array.isArray(markdown) ? markdown.join('\n') : markdown}
       unwrapDisallowed={true}
-      allowedElements={['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'strong', 'em']}
+      allowedElements={[
+        'h1',
+        'h2',
+        'h3',
+        'h4',
+        'h5',
+        'h6',
+        'p',
+        'strong',
+        'em',
+      ]}
```

Comment:
@Twiineenock AFAICT, it doesn't seem as though the allowedElements have changed per this this diff. Does that invalidate the PR title?  Is the main change here that we're just bumping `react-markdown`?

Reply:
And yes the PR title has to change! Thanks!